### PR TITLE
[pokt-network-pda] Add Pokt network strategy

### PR DESCRIPTION
**Pull request by ahmadyazdanii**
Fixes:
- None

Changes proposed in this pull request:
-  Add Pokt network strategy

### CODE REVIEW

**Review by ChaituVR — APPROVED** [review]

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Hi @ahmadyazdanii Looks like test cases are failing, can take a look? make sure to run `yarn test --strategy=pokt-network-pda` in your local

**Comment by ahmadyazdanii** [comment]
Hi @ChaituVR, Thank you for informing that.
As you said, I've fixed the problem and tested it locally. Please review it again and let me know if more work is needed.

File: src/strategies/index.ts [hunk 1]
```diff
@@ -616,7 +617,8 @@ const strategies = {
   'sunrisegaming-univ2-lp': sunriseGamingUniv2Lp,
   'sunrisegaming-staking': sunriseGamingStaking,
   'single-staking-autocompound-balanceof': singleStakingAutoCompoundBalanceOf,
-  'single-staking-longtermstaking-balanceof': singleStakingLongTermStakingBalanceOf,
+  'single-staking-longtermstaking-balanceof':
+    singleStakingLongTermStakingBalanceOf,
```
Comment (ChaituVR) [inline src/strategies/index.ts:621]:
```suggestion
  'single-staking-longtermstaking-balanceof': singleStakingLongTermStakingBalanceOf,
```

File: src/strategies/pokt-network-pda/index.ts [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+}
+
+interface SumHousesPoints {
+  builder_points: number;
+  validator_and_liquidity_provider_points: number;
+  gateway_points: number;
+}
+
+// Major constants
+export const author = 'pokt-network';
```
Comment (ChaituVR) [inline src/strategies/pokt-network-pda/index.ts:50]: @ahmadyazdanii Should be coming from `pokt-network`
s github or can change it your name

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/pokt-network-pda/index.ts (hunk 2 above)
Comment (ChaituVR) [reply src/strategies/pokt-network-pda/index.ts:50]:
```suggestion
export const author = 'ahmadyazdanii';
```

Comment (ahmadyazdanii) [reply src/strategies/pokt-network-pda/index.ts:50]: Can I use Microflow-xyz instead of my username?

Comment (ChaituVR) [reply src/strategies/pokt-network-pda/index.ts:50]: Yep for sure

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**